packages/analytics/amplify_analytics_pinpoint/lib/src/analytics_plugin_impl.dart

```diff
@@ -26,18 +25,16 @@ import 'package:meta/meta.dart';
 class AmplifyAnalyticsPinpoint extends AmplifyAnalyticsPinpointDart {
   /// {@macro amplify_analytics_pinpoint.analytics_plugin_impl}
   AmplifyAnalyticsPinpoint({
-    // TODO(fjnoyp): Rename to reflect the data that is stored.
```

Comment:
I think I had suggested this TODO be added in a previous PR. Can we give this a better name? What key/values are being stored?